deepchecks/checks/performance/regression_error_distribution.py

```diff
@@ -0,0 +1,89 @@
+"""The regression_error_distribution check module."""
+import matplotlib.pyplot as plt
+import pandas as pd
+from scipy.stats import kurtosis
+from sklearn.base import BaseEstimator
+from sklearn.metrics import mean_squared_error
+
+from deepchecks import CheckResult, Dataset, SingleDatasetBaseCheck, ConditionResult
+from deepchecks.utils.metrics import ModelType, task_type_validation
+from deepchecks.utils.strings import format_number
+
+__all__ = ['RegressionErrorDistribution']
+
+
+class RegressionErrorDistribution(SingleDatasetBaseCheck):
+    """Calculate MSE and kurtosis, display an error histogram and most extreme prediction errors.
+
+    Args:
+        n_extreme (int): amount of samples to show which are of Largest under / over estimation errors.
+    """
+
+    def __init__(self, n_extreme: int = 3):
+        super().__init__()
+        self.n_extreme = n_extreme
+
+    def run(self, dataset: Dataset, model: BaseEstimator) -> CheckResult:
+        """Run check.
+
+        Arguments:
+            dataset (Dataset): A dataset object.
+            model (BaseEstimator): A scikit-learn-compatible fitted estimator instance
+
+        Returns:
+           CheckResult:
+                - value is a dict with mse and kurtosis values.
+                - display is histogram of error distirbution and the extreme prediction errors.
+
+        Raises:
+            DeepchecksValueError: If the object is not a Dataset instance with a label
+        """
+        return self._regression_error_distribution(dataset, model)
+
+    def _regression_error_distribution(self, dataset: Dataset, model: BaseEstimator):
+        check_name = self.__class__.__name__
+        Dataset.validate_dataset(dataset, check_name)
+        dataset.validate_label(check_name)
+        task_type_validation(model, dataset, [ModelType.REGRESSION], check_name)
+
+        y_test = dataset.label_col()
+        y_pred = model.predict(dataset.features_columns())
+
+        mse = mean_squared_error(dataset.label_col(), y_pred, squared=False)
```

Comment:
So it is MSE and not RMSE? Why, just out of curiousity?

Reply:
it is rmse